### ISSUE TITLE
Instant crash with latest Homebrew (Vim 8.0.0703)

### ISSUE THREAD

**Issue by ptzz** [Shougo/neocomplete.vim#590]
# Problems summary
Crashes/freezes immediately on first character in insert mode with most recent Homebrew Vim. Bottom of the window shows:
```
-- INSERT --
line    1:
E121: Undefined variable: s:timer
E15: Invalid expression: s:timer.event
```

It works with the previous Homebrew version of Vim (8.0.0698) so possibly/probably an issue with Vim?

## Environment Information
 * OS: macOS Sierra 10.12.5
 * Vim version: v8.0.0703
```
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Jul 10 2017 21:54:55)
MacOS X (unix) version
Included patches: 1-703
Compiled by Homebrew
Huge version without GUI.  Features included (+) or not (-):
+acl             +file_in_path    +mouse_sgr       +tag_old_static
+arabic          +find_in_path    -mouse_sysmouse  -tag_any_white
+autocmd         +float           +mouse_urxvt     -tcl
-balloon_eval    +folding         +mouse_xterm     +termguicolors
-browse          -footer          +multi_byte      +terminfo
++builtin_terms  +fork()          +multi_lang      +termresponse
+byte_offset     -gettext         -mzscheme        +textobjects
+channel         -hangul_input    +netbeans_intg   +timers
+cindent         +iconv           +num64           +title
-clientserver    +insert_expand   +packages        -toolbar
+clipboard       +job             +path_extra      +user_commands
+cmdline_compl   +jumplist        +perl            +vertsplit
+cmdline_hist    +keymap          +persistent_undo +virtualedit
+cmdline_info    +lambda          +postscript      +visual
+comments        +langmap         +printer         +visualextra
+conceal         +libcall         +profile         +viminfo
+cryptv          +linebreak       +python          +vreplace
+cscope          +lispindent      -python3         +wildignore
+cursorbind      +listcmds        +quickfix        +wildmenu
+cursorshape     +localmap        +reltime         +windows
+dialog_con      +lua             +rightleft       +writebackup
+diff            +menu            +ruby            -X11
+digraphs        +mksession       +scrollbind      -xfontset
-dnd             +modify_fname    +signs           -xim
-ebcdic          +mouse           +smartindent     -xpm
+emacs_tags      -mouseshape      +startuptime     -xsmp
+eval            +mouse_dec       +statusline      -xterm_clipboard
+ex_extra        -mouse_gpm       -sun_workshop    -xterm_save
+extra_search    -mouse_jsbterm   +syntax
+farsi           +mouse_netterm   +tag_binary
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
  fall-back for $VIM: "/usr/local/share/vim"
Compilation: clang -c -I. -Iproto -DHAVE_CONFIG_H   -DMACOS_X_UNIX  -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
Linking: clang   -L. -fstack-protector -L/usr/local/lib -L/usr/local/opt/libyaml/lib -L/usr/local/opt/openssl/lib -L/usr/local/opt/readline/lib  -L/usr/local/lib -o vim        -lncurses -liconv -framework Cocoa  -L/usr/local/lib -llua -mmacosx-version-min=10.12 -fstack-protector-strong -L/usr/local/lib  -L/usr/local/Cellar/perl/5.26.0/lib/perl5/5.26.0/darwin-thread-multi-2level/CORE -lperl -lm -lutil -lc -F/usr/local/opt/python/Frameworks -framework Python   -lruby.2.4.1 -lobjc
```

## Provide a minimal .vimrc with less than 50 lines (Required!)

```vim
 call plug#begin('~/.vim/plugged')
 Plug 'Shougo/neocomplete'
 call plug#end()
 let g:neocomplete#enable_at_startup = 1
```

## The reproduce ways from Vim starting (Required!)

 1. Start vim
 2. Enter insert mode
 3. Type any character

**Comment by Shougo** [comment]
Vim 8.0.702 is broken.

https://groups.google.com/forum/#!topic/vim_dev/l5JhLQB_5xA/discussion

**Comment by Shougo** [comment]
Fixed in 8.0.705.
